tests/callbacks/test_plotting_callback.py

```diff
@@ -77,6 +81,35 @@ def test_errorbars_created_if_yerr_is_given():
     ax.errorbar.assert_called_with(y=[y], x=[x], yerr=[yerr], fmt="none")
 
 
+def test_png_saved_on_run_stop():
+    ax = MagicMock(spec=Axes)
+    ax.figure = MagicMock(spec=Figure)
+    s = PlotPNGSaver(x="x", y="y", ax=ax, postfix="123", output_dir="")
+
+    s.stop(
+        {
+            "data": {
+                "x": 1,
+                "y": 2,
+            },  # pyright: ignore [reportArgumentType]
+            "rb_number": 1234,  # pyright: ignore [reportArgumentType]
+            "time": 123456789,
+            "uid": "1",
+            "exit_status": "success",
+            "run_start": "",
+        }
+    )
```

Comment:
`pyright` is correct here, neither `data` nor `rb_number` belong in a stop document.